src/build/mkrules/cflags.env.mk

```diff
@@ -41,7 +41,7 @@ CFLAGS += $(COMMONFLAGS) -mcpu=power7 -nostdinc -g -mno-vsx -mno-altivec\
 	  -ffunction-sections -fdata-sections -ffreestanding -mbig-endian
 ASMFLAGS += $(COMMONFLAGS) -mcpu=power7 -mbig-endian
 CXXFLAGS += $(CFLAGS) -nostdinc++ -fno-rtti -fno-exceptions -Wall \
-	    -fuse-cxa-atexit
+	    -fuse-cxa-atexit -std=gnu++11
```

Comment:
This should be -std=gnu++03, not -std=gnu++11.  Hostboot P8 is not C++11 compliant and there is enough drastically different between C++03 and C++11 that I would not trust running Hostboot P8 compiled with C++11.  I did a number of changes for P9 to support C++11 properly.